packages/forms-web-app/src/routes/before-you-start/enforcement-notice.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+
+const fetchExistingAppealMiddleware = require('../../middleware/fetch-existing-appeal');
+const enforcementNoticeController = require('../../controllers/before-you-start/enforcement-notice');
+const { validationErrorHandler } = require('../../validators/validation-error-handler');
+const {
+  rules: enforcementNoticeValidationRules,
+} = require('../../validators/before-you-start/enforcement-notice');
+
+const router = express.Router();
+
+router.get(
```

Comment:
Isn't this functionality included in the use-a-different-notice ticket?

Reply:
Removed